### ISSUE TITLE
User wants to know text is going to appear correctly

### ISSUE THREAD

**Issue by sonyagreen** [resin-io/pensieve#167]
In some cases (eg. when writing a lot, writing markdown, or filling in lots of fields) user wants to ensure that text doesn't disappear, and will show up correctly.

Today when testing Pensieve, @lekkas and I ran into the issue of wanting to fill in an entry half-way, then save it before continuing. It's that "I've put enough work into this that if I accidentally cancelled this entry or my computer crashed I would cry" impulse. 

We wished for autosave in the browser, or a preview to show how markdown rendered. Each save along the way required a commit message, which slowed the ability to notice markdown mistakes and fix them.

**Comment by lurch** [comment]
Yeah, even without autosave, a Markdown Preview button / tab like GitHub uses would be very useful :)
